input/pagecontent/general-requirements.md

```diff
@@ -344,6 +344,7 @@ AU Core Diagnostic Result Observation|Observation.performer|AU Core Practitioner
 AU Core Diagnostic Result Observation|Observation.hasMember|AU Core Diagnostic Result Observation, AU Core Pathology Result Observation
 AU Core Pathology Result Observation|Observation.performer|AU Core Practitioner, AU Core PractitionerRole, AU Core Organization, AU Core Patient, AU Base RelatedPerson
 AU Core Procedure|Procedure.reasonReference|AU Core Condition, Observation, AU Core Procedure, DocumentReference
+AU Core MedicationRequest|MedicationRequest.reasonReference|AU Core Condition, Observation
 {:.grid}
```

Comment:
Missing the addition of MedicationRequest to "Must Support - Choice of Profile Elements".